charts/datadog/values.yaml

```diff
@@ -256,6 +256,24 @@ datadog:
   # datadog.criSocketPath -- Path to the container runtime socket (if different from Docker)
   criSocketPath:  # /var/run/containerd/containerd.sock
 
+  # datadog.serviceTopology -- Configure service topology to send custom metrics and traces without using host ports
+  # Important notes:
+  # - The Service Topology feature in Kubernetes is still in alpha and disabled by default, please make sure it's enabled in your cluster configuration
+  # - The environment variable DD_AGENT_HOST in your application pod template must be configured to reach the topology service
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service-topology/
+  serviceTopology:
+
+    # datadog.serviceTopology.enableForDogstatsd -- Enabling this will allow sending custom metrics to the Datadog Agent on the same node without using a host port
+    enableForDogstatsd: false
+
+    # datadog.serviceTopology.enableForAPM -- Enabling this will allow sending APM traces to the Datadog Agent on the same node without using a host port
+    enableForAPM: false
```

Comment:
I'm wondering if we should put it under `datadog.apm` like `datadog.apm.serviceTopologyEnabled`
same for dogstatsd.

Reply:
Thought about it as well, I found it easier for customers to regroup the service config under the same field, if enable service topology there is a good chance you want for both apm and dsd right?